Editor/BacktraceConfigurationEditor.cs

```diff
@@ -93,6 +92,13 @@ public override void OnInspectorGUI()
                     EditorGUILayout.HelpBox("Please insert value greater or equal -1", MessageType.Error);
                 }
             }
+
+#if UNITY_ANDROID || UNITY_IOS
+            EditorGUILayout.PropertyField(
+                serializedObject.FindProperty("AttachmentPaths"),
+                new GUIContent(BacktraceConfigurationLabels.LABEL_REPORT_ATTACHMENTS));
```

Comment:
@konraddysput does this end up exposing the array as we discussed in the editor?  (The default foldout arrow thing?)  If so, this is good for now!  Side note - as a part of the work I am doing, I am going to try to come up with some custom property drawers to make these easier to use

Reply:
Yes it does - I hope we can make it prettier soon!